big_tests/tests/login_SUITE.erl

```diff
@@ -219,6 +272,29 @@ configure_sha384_fail_log_with_sha512(Config) ->
 configure_sha512_fail_log_with_sha1(Config) ->
     configure_and_fail_log_scram(Config, sha512, <<"SCRAM-SHA-1">>).
 
+%%
+%% configure_sha*_plus_fail_log_with_sha* tests are succeeding due to the fact that
+%% escalus, when login with scram, sets channel binding flag to 'y'. This indicates that
+%% escalus supports channel binding but the server does not. The server did advertise the
+%% SCRAM PLUS mechanism, so this flag is incorrect and could be the result of the
+%% man-in-the-middle attack attempting to downgrade the authentication mechanism.
+%% Because of that, the authentication should fail.
+%%
```

Comment:
@michalwski kinda like that, yeah. Server and Client must agree on the highest common authentication mechanism. If the client tells to the server that it supports `-PLUS` and then it doesn't choose it, the server must interpret that as a downgrade attack (https://tools.ietf.org/html/rfc5802#page-15)

Reply:
Yes, you are correct. 

The way for other logins is achieved is that for other auth mechanisms the users are not using fast_tls and -PLUS mechanisms are not advertised to them. 

When client supports -PLUS and server does not advertise one, it will set the channel binding flag to `y`. What I wanted to mark here is that escalus is not analysing on what is being advertised, it will either use channel binding with flag `p` or send `y` flag regardless of what was advertised. This could potentially lead to failing authentications in tests where we do advertise -PLUS but try to use other SCRAM method. I think it is not clearly visible that this could happen, hence decided to leave a note that one day could be a clue on failing test.

Reply:
Thanks for clarifications!